.pre-commit-config.yaml

```diff
@@ -13,3 +13,8 @@ repos:
         additional_dependencies:
           - "types-pytz"
           - "types-requests"
+
+  - repo: "https://github.com/python-poetry/poetry"
+    rev: "1.2.0b3"
```

Comment:
Let's switch to stable release before merging

Reply:
@hovaesco the reason for including a pre-release it it exposes [dependency groups](https://python-poetry.org/docs/master/managing-dependencies/). It's not clear from https://github.com/python-poetry/poetry/issues/5586 when the 1.2.0 release is scheduled.

Are there any major concerns with using a pre-release in the interim especially if it seems to fulfill our needs?